extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFieldsMixin.json

```diff
@@ -0,0 +1,92 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFields",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Orchestration Step Event Action Execution Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEvent",
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "This Mixin will have Journey Action Metadata Fields",
+  "definitions": {
+    "journeyStepEventActionExecutionFields": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/stepEvents": {
+          "title": "Journey Orchestration for creating user journey.",
+          "type": "object",
+          "properties": {
```

Comment:
rather than `journey:actionXXX` it would be cleaner to have `journeyAction:XXX` or even better - just `journey:` or even `action`

Reply:
This is the custom action one can create and we thought to make it work like this with the journey:actionId, so during provisioning will only require limited namespace creation. please let us know if it's fine.

Reply:
I don't understand how the specific namespace would matter in terms of provisioning...Why not do it correctly?

Reply:
@nikshar  - In this PR you are trying to introduce a new shorthand namespace context "journey". This is fine, but then you need to also update the common/context.jsonld file with what "journey" maps to. 

If the goal is to use this schema in AEP, I would avoid the shorthand "journey" and simply use the URI form for all your field (keep the parent and child namespaces the same). "journey:actionId" would become "https://ns.adobe.com/experience/journeyOrchestration/actionId". This will make it so in AEP your filed path will be "_experience.journeyOrchestration.stepsEvents.actionId". If you keep it as it is now, your AEP path will be ""_experience.journeyOrchestration.stepsEvents._journey.actionId"

Reply:
Thanks @cmathis ! It explains a lot.
I've updated the mixins with the changes please have a look.

Reply:
_experience.journeyOrchestration.stepEvents._experience. journeyOrchestration. actionId - This path is hard to digest. Not recommended at all.

Please change for better visibility and clear understanding.

Reply:
Hi @vikas, 
right now it's being changed as per Craig's comments, are you referring here becausee the whole path due to parent step as well 
 ```"journeyStepEventActionExecutionFields": {
      "properties": {
        "https://ns.adobe.com/experience/journeyOrchestration/stepEvents": {
          "title": "Journey Orchestration Action Execution Fields.",
          "type": "object",
          "properties": {
            "https://ns.adobe.com/experience/journeyOrchestration/actionId":{
          ```
should wee move back to orriginal journye:actionID or just actionId .
sorry may be missing something here

Reply:
I don't think you want the /stepEvents/ in your sub-fields - this gives it a completely different namespace than the parent field, but we can try this.

Reply:
All fields should be namespaced with just "https://ns.adobe.com/experience/journeyOrchestration/"